src/turnkeyml/cli/cli.py

```diff
@@ -472,26 +469,28 @@ def check_extension(choices, file_name):
     # on a target script. If the user doesn't provide a command,
     # we alter argv to insert the command for them.
 
+    # Special characters that indicate a string is a filename, not a command
+    file_chars = [".", "/", "\\", "*"]
```

Comment:
We used on only allow `.` to identify files, which would have excluded something like `timm/*` on Windows.